modules/rhcos-enabling-multipath.adoc

```diff
@@ -1,122 +1,65 @@
-----
-
-. Create the new machine config by using either the master or worker YAML file you previously created:
+. To enable multipath and start the `multipathd` daemon, run the following command:
 +
 [source,terminal]
 ----
-$ oc create -f ./99-master-kargs-mpath.yaml
+$ mpathconf --enable && systemctl start multipathd.service
 ----
+** Optional: If booting the PXE or ISO, you can instead enable multipath by adding `rd.multipath=default` from the kernel command line.
 
-. Check the machine configs to see that the new one was added:
+. Append the kernel arguments by invoking the `coreos-installer` program:
 +
-[source,terminal]
-----
-$ oc get MachineConfig
-----
+* For a single multipath device, use the path `/dev/mapper/mpatha`, for example:
 +
-.Example output
 [source,terminal]
 ----
-NAME                                               GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
-00-master                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-00-worker                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-master-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-master-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-kargs-mpath                              52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             105s
-99-master-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-ssh                                                                                 3.2.0             40m
-99-worker-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-worker-ssh                                                                                 3.2.0             40m
-rendered-master-23e785de7587df95a4b517e0647e5ab7   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-rendered-worker-5d596d9293ca3ea80c896a1191735bb1   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
+$ coreos-installer install /dev/mapper/mpatha \ <1>
+--append-karg rd.multipath=default \
+--append-karg root=/dev/mapper/dm-mpath-root
 ----
-
-. Check the nodes:
+<1> Indicates the path of the single multipathed device.
 +
-[source,terminal]
-----
-$ oc get nodes
-----
+* For multiple multipath devices, or to be more explicit, it is recommended that the names of the multipath devices are aliased to a World Wide Name (WWN) symlink when you append the kernel arguments , such as `/dev/disk/by-label`. For example:
```

Comment:
```suggestion
* For multiple multipath devices, or to be more explicit, instead of using e.g. `/dev/mapper/mpatha`, it is recommended to use the World Wide Name (WWN) symlink available in `/dev/disk/by-id`. For example:
```

Reply:
Resolved